src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsDetection.java

```diff
@@ -10,13 +10,15 @@
 public class TestAdsDetection extends NewTestTemplate {
 
   public static final String PIXEL_PATTERN_WITH_ADBLOCK =
-      "http://www\\..*\\.com/bcn.*deo=1.*";
+      "https?://www\\..*\\.com/bcn.*deo=1.*";
   public static final String PIXEL_PATTERN_WITHOUT_ADBLOCK =
-      "http://www\\..*\\.com/bcn.*deo=0.*";
+      "https?://www\\..*\\.com/bcn.*deo=0.*";
+//  public static final String PIXEL_PATTERN_WITHOUT_ADBLOCK =
+//          ".*acdn.*";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)